packages/sanity/src/structure/panes/documentList/sheetList/useDocumentSheetColumns.tsx

```diff
@@ -1,70 +1,33 @@
-import {isObjectSchemaType, type ObjectSchemaType} from '@sanity/types'
-import {Box, Checkbox, Flex, Text} from '@sanity/ui'
+import {type IntrinsicTypeName, isObjectSchemaType, type ObjectSchemaType} from '@sanity/types'
+import {Checkbox, Flex} from '@sanity/ui'
 import {
   type AccessorKeyColumnDef,
   createColumnHelper,
   type GroupColumnDef,
   type VisibilityState,
 } from '@tanstack/react-table'
 import {useMemo} from 'react'
-import {useMemoObservable} from 'react-rx'
-import {
-  type DocumentPreviewStore,
-  DocumentStatusIndicator,
-  getPreviewStateObservable,
-  type SanityDocument,
-  type SchemaType,
-  useDocumentPreviewStore,
-} from 'sanity'
-
-import {type PaneItemPreviewState} from '../../../components/paneItem/types'
+
 import {DocumentSheetListSelect} from './DocumentSheetListSelect'
+import {PreviewCell} from './DocumentSheetPreviewCell'
 import {SheetListCellInner} from './SheetListCell'
+import {type DocumentSheetTableRow} from './types'
 
 export const VISIBLE_COLUMN_LIMIT = 5
 
-const PreviewCell = (props: {
```

Comment:
This was moved to it's own file https://github.com/sanity-io/sanity/pull/6889/files#diff-aa2b7a99d4136b04a6697b70f21fa1e644434bc498de1fdab03527fd43140084